sw/device/lib/dif/BUILD

```diff
@@ -37,7 +37,7 @@ cc_library(
     ],
     deps = [
         ":base",
-        "//hw/ip/adc_ctrl/data:adc_ctrl_c_regs",
```

Comment:
Same as for lib/crypto/drivers. These are very regular rules for dif, drivers, and silicon_creator/drivers (at least).